### DISCUSSION TITLE
Atributo "Height (U)" é inválido para Rack

### DISCUSSION THREAD

**Discussion by mateeussmartiins** [netbox-community/netbox#17862] [Help Wanted!]
ESSA PARTE SEMPRE APRESENTA ERRO , AIDA NÃO SEI O MOTIVO.
ALGUEM PODE AJUDAR.

**Comment by betadrome** [comment]
> ESSA PARTE SEMPRE APRESENTA ERRO , AIDA NÃO SEI O MOTIVO. ALGUEM PODE AJUDAR.

@mateeussmartiins, can you provide more information next time to reproduce the error, also in English if it's possible? I think the problem occurs when creating a rack? I can reproduce the error in "demo.netbox.dev" but not in my own netbox, both on version 4.1.4. Maybe the problem will be solved until the demo data has been regenerated (afaik every 24h).

**Reply by mateeussmartiins** [reply]
Attribute "Height (U)" is invalid for Rack

![image](https://github.com/user-attachments/assets/15d3dbb7-6edf-4eff-bb4b-6fabb2aa6d10)


sending an error image as an attachment. I've already changed all the shapes. I believe it's a bug in the platform that doesn't allow you to add a rack

**Reply by mateeussmartiins** [reply]
I'm trying to add an interface between SW and Patch Pannel, but I can't find an interface, even registered as shown in the image.

![image](https://github.com/user-attachments/assets/77b031b6-8b46-4e04-99db-1152b71972d6)

![image](https://github.com/user-attachments/assets/ba7f97c6-0f25-4b69-8070-4cab0b760a2f)

**Comment by betadrome** [comment]
@mateeussmartiins, ok I found the "problem". In the demo instance there was a custom validation set for racks:

```text
{
    "dcim.rack": [
        {
            "Height (U)": {
                "maximum_value": 200
            }
        }
    ]
}
```
I removed this configuration and rack creation working fine. Since version 4, some validator names have changed, which is why the validator described above no longer works. Here is the working version which works:

```text
{
    "dcim.rack": [
        {
            "u_height": {
                "max": 200
            }
        }
    ]
}
```

**Reply by mateeussmartiins** [reply]
OK, solved it. but I have another problem. This description field is mandatory, but it is not possible to edit


![image](https://github.com/user-attachments/assets/559efa97-e519-4bd2-bc5c-54b39897f596)

**Reply by candlerb** [reply]
That is not a "description field".

"Portas traseiras" I presume means "Rear ports". If you are trying to create a front port, then you *must* associate it with a rear port; this is required so that paths can be traced through patch panels.

Multiple front ports can be associated with the same rear port, if the rear port has multiple "positions" (i.e. strands). But you can't create a front port that's not linked to any rear port.

This means in turn that you must create rear ports on your device, before you can create the corresponding front ports.

**Reply by mateeussmartiins** [reply]
I UNDERSTAND, BUT THE REAR DOORS FIELD IS A DESCRIPTION FIELD. THE MOUSE IS LOCKED.

I ALREADY HAVE THE FIELDS FOR THE REAR AND FRONT DOORS, BUT THE FIELD REMAINS LOCKED.

**Reply by candlerb** [reply]
What version of Netbox are you using?

On a current version of Netbox it's not a description field. It's a selection field, where you select between existing rear ports on the device. Until you have created some rear ports on the device, you won't have any values to select.

For example, if you look on demo.netbox.dev (v4.1.5):

https://demo.netbox.dev/dcim/devices/88/front-ports/
https://demo.netbox.dev/dcim/front-ports/1297/edit/

![image](https://github.com/user-attachments/assets/63a76bd1-bba0-4da4-9f94-557a232ac531)
